src/hostnet/hostnet_http.ml

```diff
@@ -282,10 +282,22 @@ module Make
                     (Cohttp.Response.sexp_of_t res)));
       let reader = Outgoing.Response.make_body_reader res outgoing in
       Incoming.Response.write ~flush:true (fun writer ->
-          match Incoming.Response.has_body res with
-          | `Yes     -> proxy_body_response ~reader ~writer
-          | `No      -> Lwt.return_unit
-          | `Unknown ->
+          match Cohttp.Request.meth req, Incoming.Response.has_body res with
+          | `HEAD, _ ->
```

Comment:
Should this be `HEAD, Yes`? Since `HEAD, No`, is the expected state, isn't it?

Reply:
Yeah that would be clearer -- so in that case the first clause would be specific to the bug and match the comment more closely. I'll change that.